test.js

```diff
@@ -0,0 +1,252 @@
+import test from 'node:test'
```

Comment:
I didn't think much about the test framework to use, so went with the simplest solution I can think of